File: src/Visuals.cs
using System;
using System.Threading;

class Visuals
{
    public static void Header()
    {
        String[] i = new String[4];
        i[0] = "\n**THE PLAINS: REBOOT**************";
        i[1] = "**MADE BY DRAUMAZ IN 2021*********";
        i[2] = "**WRITTEN IN C#!******************";
        i[3] = "**CHARACTER DESIGN BY BRYCE CANO**\n";
        foreach (String s in i)
        {
            Console.WriteLine(s);
        }
    }
    public static void Exit_Header()
    {
        String url = "https://github.com/draumaz/plains-reboot";
        Tools.Clear_Screen();
        Console.WriteLine("\nThanks for playing my game!");
        Tools.Snooze(500);
        Console.Write("\nKeep up with development at ");
        Read_Looper(url, 10);
        Console.Write(".\n");
        Tools.Snooze(200);
    }
    public static void Read_Looper(String s, int speed)
    {
        String[] x = Tools.String_Array_Inator(s);
        for (int i = 0; i < x.Length; i++)
        {
            Console.Write(x[i]);
            Tools.Snooze(speed);
        }
    }
}

File: src/visuals.py
from tools import version, clear, sleep, is_win
from reader import readers

class headers:
    def splash_header():
        print("\n===========THE PLAINS=============")
        print("=====MADE BY DRAUMAZ IN 2021======")
        print("=======WRITTEN IN PYTHON!=========")
        print("==CHARACTER DESIGN BY BRYCE CANO==\n")
    def version_header():
        print("\nThe Plains", version(), "\n", end="")
    def exit_header():
        clear()
        print("\nThanks for playing my game!")
        sleep(0.5)
        print("\nKeep up with development at ", end="")
        readers.array_reader("https://github.com/draumaz/plains", 0.004)
        print(".\n")
        sleep(0.05)
        if is_win() == 1:
            input("Press ENTER to continue: ")
            print()